example_client/multi_inputs.py

```diff
@@ -0,0 +1,279 @@
+# -f /var/repos/github/sample-videos/person-bicycle-car-detection.mp4
+# SSDMobileNet: python3.6 multi_inputs.py -n "SSDMobileNet" -l "image_tensor"
+# -o "DetectionOutput" -d 300 -i 127.0.0.1 -p 9001 -c 1
+# -f /var/repos/github/sample-videos/person-bicycle-car-detection.mp4
+
+from __future__ import print_function
+from argparse import ArgumentParser, SUPPRESS
+from tensorflow_serving.apis import predict_pb2
+from tensorflow_serving.apis import prediction_service_pb2_grpc
+from time import time, sleep
+
+import sys
+import os
+import cv2
+import grpc
+import threading
+import numpy as np
+import logging as log
+import tensorflow.contrib.util as tf_contrib_util
+
+# global data (shared between threads & main)
+CLASSES = ["None", "Pedestrian", "Vehicle", "Bike", "Other"]
+COLORS = [(255, 255, 255), (255, 0, 0), (0, 255, 0), (0, 0, 255), (128, 128, 128)]
+SRC_TYPE = ["Camera", "Video"]
+
+exit_ok = False         # manage thread loop
+cam_width = 640         # camera width
+cam_height = 480        # camera height
+cam_fps = 30            # camera speed
+
+#####################################################################################
+
+def build_argparser():
+  parser = ArgumentParser(add_help=False)
+  args = parser.add_argument_group('Options')
+  args.add_argument('-h', '--help', action='help', default=SUPPRESS,
+        help='Show this help message and exit.')
+  args.add_argument('-n', '--network_name', required=True,
+        type=str, help='Network name')
+  args.add_argument('-l', '--input_layer', required=True,
+        type=str, help='Input layer name')
+  args.add_argument('-o', '--output_layer', required=True,
+        type=str, help='Output layer name')
+  args.add_argument('-d', '--input_dimension', required=True,
+        type=int, help='Input image dimension')
+  args.add_argument('-c', '--num_cameras', help='Number of cameras to be used',
+        required=False, type=int, default=1)
+  args.add_argument('-f', '--file', help='Path to the video file',
+        required=False, type=str)
+  args.add_argument('-i', '--ip', help='ip address of the ovms', required=True)
+  args.add_argument('-p', '--port', help='port of the ovms', required=True)
+
+  return parser
+
+# Decoding idea based on the link below. Not very accurate. So pls implement yours
+# https://github.com/opencv/open_model_zoo/blob/master/intel_models/\
+# person-vehicle-bike-detection-crossroad-0078/\
+# description/person-vehicle-bike-detection-crossroad-0078.md
+def parse_output(thr_id, res, frame):
+  for batch, data in enumerate(res):
+    pred = data[0]
+    for values in enumerate(pred):
+      # tuple
+      index = values[0]
+      l_pred = values[1]
+
+      # actual predictions
+      img_id = l_pred[0]
+      label = l_pred[1]
+      conf = l_pred[2]
+      x_min = l_pred[3]
+      y_min = l_pred[4]
+      x_max = l_pred[5]
+      y_max = l_pred[6]
+
+      # preventing any wrong array indexing (for RMNet)
+      if label >= 4:
```

Comment:
If `l_pred[1]` is 4, then you have` label = 4 `which I believe is okay, but then it gets into this conditional block, logs error and again assigns `label = 4`. It might be confusing. 
I suggest changing this condition to `if label > 4`

Reply:
Done